src/machine/machine_rp2040_xosc.go

```diff
@@ -30,7 +30,8 @@ func (osc *xoscType) init() {
 	osc.ctrl.Set(rp.XOSC_CTRL_FREQ_RANGE_1_15MHZ)
 
 	// Set xosc startup delay
-	delay := (((xoscFreq * MHz) / 1000) + 128) / 256
+	multiplier := 64
+	delay := (((xoscFreq * MHz) / 1000) + 128) / 256 * multiplier
```

Comment:
Changing `((xoscFreq * MHz) / 1000)` to `xoscFreq * KHz + 128` can be potentially confusing since `xoscFreq` is defined as 12 (MHz) in the various board files.